v3/src/components/graph/components/graph.tsx

```diff
@@ -31,105 +32,133 @@ interface IProps {
   dotsRef: React.RefObject<SVGSVGElement>
 }
 
-const marqueeState = new MarqueeState()
+const marqueeState = new MarqueeState(),
+  dataTip = d3tip().attr('class', 'graph-d3-tip')/*.attr('opacity', 0.8)*/
+    .html((d: string) => {
+      return "<p>" + d + "</p>"
+    })
```

Comment:
Seems like we might have to worry about sanitization here, e.g. if the content being rendered included `</p>` then we'd end up with invalid html. Doesn't necessarily have to be addressed in this PR, but perhaps a `TODO` comment to remind us to come back to it would be warranted.